ota-plus-web/docs/modules/ROOT/pages/use-case-ecm.adoc

```diff
@@ -0,0 +1,275 @@
+
+== A new release is announced
+
+Your release manager contacts you and says that they want to update the software on the Engine Control Module (ECM). 
+
+* The goal is to extend the engine idle time as much as possible to allow the Heating, ventilation, and air conditioning (HVAC) to keep the driver comfortable during extreme weather conditions. 
+* The update will override engine shutdown if temperature falls below 7.2 degrees Celsius or above 35 degrees Celsius.
+
+They have the following release conditions:
+
+* The release only applies to editions of the Lannista "Tireon" model built after 2018
+* Restrict the update to the Scandinavian market 
+* Target only customers who have opted for premium support.
+
+== Device groups
+
+=== Preparation
+You can run detailed search queries in CarHQ and export the data to a CSV file. Your search query might look something like this. It's simplified for the purposes of this tutorial.
+
+[source,sql]
+----
+Select VIN
+FROM CustomerView
+WHERE model="Tireon" AND "BuildDate => 2018-01-01" AND Region IN ('DK, NO, SE, FI') AND SupportLevel ="010A"
+----
+
+In this case, you'd want to create a fixed group. 
+
+* A smart group is not sophisticated enough for your query since smart groups can only filter by Device ID.
+* Instead, you can use a fixed group and import the search results that you exported from CarHQ.
+
+// MC: Hack to make this section collapsible
++++ <details><summary> +++
+*What should have happened first* 
++++ </summary><div> +++
+
+To group your vehicles, they must already exist as "devices" in OTA Connect. In other words, your developers should have provisioned these devices with a proper device identifier - in this case we're using the VIN number.
+
++++ </div></details> +++
+
+For the sake of this use case, lets say you exported a file called "__Tireon-vins.txt__" from carHQ.
+
+*To create the device group, you'd follow these steps*
+
+. In the OTA Connect Portal, navigate to the *Device groups* page and click **Create device group**.
+. On the *Create group* page, make sure that *Fixed group* is selected.
+. Enter the name like "__Tireon-2018-Scandinavia-010A__" for the group name.
+. In the *Optional Import* section, click *Choose file* and select the "__Tireon-vins.txt__" file that you saved previously.
+. Click **Create**.
++
+You should see matching devices show up in your group. For simplicity's sake, let's say that the group contains exactly 1,000 devices. This is useful to know when we track the progress of our software updates.
+
+== Preparing the software update
+
+// MC: Hack to make this section collapsible
++++ <details><summary> +++
+*What should have happened first* 
++++ </summary><div> +++
+
+Your developers have bitbaked a "baseline" disk image - a baseline disk image that runs application software which is compatible with your ECU's hardware. In this case, it's "NXP-AUTOSAR-Classic-4.2".
```

Comment:
Missed this detail before: bitbake is a yocto-specific build tool; if it's an autosar ECU it definitely wasn't created by bitbake.

Reply:
Fixed in 1aefef24a0c105565faca2b2af72e3ba034a25dc